cores/ckan_portal/conf/schema.xml

```diff
@@ -146,23 +173,13 @@
     <dynamicField name="res_extras_*" type="text" indexed="true" stored="true" multiValued="true"/>
     <dynamicField name="vocab_*" type="string" indexed="true" stored="true" multiValued="true"/>
     <dynamicField name="*" type="string" indexed="true"  stored="false"/>
-    
-    <!-- Extra Canada fields -->
-    <field name="catalog_type" type="string" indexed="true" stored="true" multiValued="true"/>
```

Comment:
I think these are used in the ckan search, why are they removed here?

Reply:
Yeah there were a fair amount of solr schema changes. So will have to add these back in. I think I am most(ish) worried about the `core_ati` changes if you are able to check that one out?? Thanks!